controllers/home.js

```diff
@@ -1,15 +1,18 @@
 "use strict";
 
+const TIPS = require("../tips");
+
 
 function home(req, res) {
-  let breach = null;
+  let featuredBreach = null;
   if (req.query.breach) {
     const reqBreachName = req.query.breach.toLowerCase();
-    breach = req.app.locals.breaches.filter(breach => breach.Name.toLowerCase() === reqBreachName)[0];
+    featuredBreach = req.app.locals.breaches.filter(breach => breach.Name.toLowerCase() === reqBreachName)[0];
```

Comment:
Should this use `.find()` a la https://github.com/mozilla/blurts-server/blob/5cb8a1cc8467dd45b0201c05b2729184fcfad2cf/controllers/hibp.js#L12